backend-api/src/functions/asyncActiveSession/jwe/symmetricDecrypter.ts

```diff
@@ -1,13 +1,13 @@
 import { createDecipheriv } from "node:crypto";
 
 export class SymmetricDecrypter implements IDecryptSymmetric {
-  async decrypt(
+  decrypt(
```

Comment:
No need for this to be async 🥹

Reply:
haha - I'd like an async symmetric decrypter